commons/zenoh-protocol/src/core/mod.rs

```diff
@@ -77,21 +77,60 @@ impl ZenohId {
         self.0.to_le_bytes()
     }
 
-    pub fn rand() -> ZenohId {
-        ZenohId(uhlc::ID::rand())
+    pub fn rand() -> ZenohIdProto {
+        ZenohIdProto(uhlc::ID::rand())
     }
 
     pub fn into_keyexpr(self) -> OwnedKeyExpr {
         self.into()
     }
 }
 
-impl Default for ZenohId {
+impl Default for ZenohIdProto {
     fn default() -> Self {
         Self::rand()
     }
 }
 
+/// The global unique id of a zenoh peer.
```

Comment:
I would expect this type to be defined in the API and not in `zenoh-protocol`.

Reply:
see below